crates/simple-git/Cargo.toml

```diff
@@ -13,11 +13,18 @@ license = "Apache-2.0 OR MIT"
 [dependencies]
 compact_str = "0.7.0"
 derive_destructure2 = "0.1"
-gix = { version = "0.52.0", features = ["blocking-http-transport-reqwest"] }
 thiserror = "1.0.40"
 tokio = { version = "1.30.0", features = ["rt", "time"], default-features = false }
 tracing = "0.1.37"
 
+[dependencies.gix]
+version = "0.53.1"
+default-features = false
+features = [
```

Comment:
I realize that with the removal of the `max-performance-safe`, also the `parallel` feature was removed. This turns thread-safe types into ones that aren't thread-safe anymore for programs that are single-threaded and don't want to pay for thread-safe structures.

Let me add a new `parallel` feature (or something similar) that makes it easy to turn that on without needing `max-performance-safe`.

Reply:
@NobodyXu I will create a release once [the PR](https://github.com/Byron/gitoxide/pull/1022) is green. Then there will be a `parallel` feature which turns thread-safety back on.

Reply:
I realize that this release would be a minor version bump once again due to some potentially breaking changes, and I wonder if you could pull in `gix-features` for a couple of weeks and activate the `parallel` feature there. Then this will be fixed with the next regular release. Alternatively, you could enabled `max-performance-safe` like before, and downgrade that to `parallel` once available.
Thanks for your consideration.

Reply:
I would probably enable max-performance-safe

Reply:
Yes, that will work. By now I don't even recommend anymore to enable it as it's a choice that affects downstream applications who then can't get pure-Rust builds anymore. But here, and for the time being, I think it's a practical choice.